app/models/team.rb

```diff
@@ -184,11 +185,21 @@ def disallow_duplicate_members
   end
 
   def limit_number_of_students
-    students = roles.select{|r| r.name == 'student' && !r.marked_for_destruction?}
+    students = members_with_role 'student'
     return unless students.size > 2
     errors.add(:roles, 'there cannot be more than 2 students on a team.')
   end
 
+  def limit_number_of_coaches
+    coaches = members_with_role 'coach'
+    return unless coaches.size > 4
+    errors.add(:roles, 'there cannot be more than 4 coaches on a team.')
```

Comment:
Can you please add the error to the locale? 
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/3fd55ebe3b221273bb0112bdbcb0cae783e6c471/config/locales/en.yml#L86-L91

Reply:
See https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/878

Reply:
Thanks for the feedback! I'm on it

Reply:
Hey @F3PiX, I've added the localisation as discussed. 

I've left it out for the student limit error, as I'm not sure it makes sense to add it to this PR. 

What do you think?

Reply:
I totally agree.